packages/document/main-doc/docs/zh/guides/basic-features/deploy.mdx

```diff
@@ -4,20 +4,23 @@ sidebar_position: 15
 
 # 部署
 
-目前 Modern.js 支持在 node.js 环境下运行，你可以自行托管，同时，Modern.js 官方支持在 Netlify 平台上部署。
+目前，Modern.js 提供了两种部署方式：
```

Comment:
The main reason for this is to emphasize that only node.js environments are supported, even on the netlify and vercel platforms, since they also support edge environments. Now this information may be missing

Reply:
I saw a block below, it will emphasize this info..